src/test/suite/fixtures/recorded/languages/rust/chuckNameDrum.yml

```diff
@@ -0,0 +1,38 @@
+languageId: rust
+command:
+  spokenForm: chuck name drum
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: d}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: name}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    let user = User {
+        first_name: "John",
+        last_name: "Doe",
+    };
+  selections:
+    - anchor: {line: 3, character: 2}
+      active: {line: 3, character: 2}
+  marks:
+    default.d:
+      start: {line: 2, character: 16}
+      end: {line: 2, character: 19}
+finalState:
+  documentContents: |
+    let user = User {
+        first_name: "John",
+        : "Doe",
```

Comment:
As mentioned in https://github.com/cursorless-dev/cursorless/pull/828#discussion_r913665423, I'd remove the leading `:` here.  I'd also remove trailing `:` when they say "chuck name" or "chuck key".  I know this looks awkward with this code, but in the following code it's perfectly reasonable:

```rust
let user = User {
        first_name: "John",
        last_name: last_name,
}
```

In this case you could say either "chuck key" or "chuck value" to switch to using [field initialiser shorthand](https://rust-lang.github.io/rfcs/1682-field-init-shorthand.html)

Reply:
How come this one still has the leading `:`?

Reply:
Ah looks like you just forgot to delete this test; I see it's failing as expected